internal/networkpolicy/reconciler.go

```diff
@@ -0,0 +1,255 @@
+package networkpolicy
+
```

Comment:
we are keeping all the reconcilers inside `controllers` folder under different packages. I think we should move this there to keep it consistent

Reply:
as per @bharath-avesha controllers which are not handling CRDs should be in internal/